CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 #### Fixes
 
+* [#1427](https://github.com/ruby-grape/grape/pull/1427): Detect content type in case of raising Rack level errors - [@iCEAGE](https://github.com/iCEAGE)
```

Comment:
Missing a period at the end.

I think the description is a bit misleading. Your problem is content type, but what is being fixed is about handling an invalid query string when trying to detect the response format, no?